FoodHaus_App/project-swift/Controller/Menu/CheckViewController.swift

```diff
@@ -8,12 +8,29 @@ class CheckViewController: UIViewController {
     
     @IBOutlet weak var tax: UILabel!
     
+    @IBOutlet weak var finalTotal: UILabel!
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        if let totalPriceText = totalPrice.text,
+        if let subTotalPriceText = totalPrice.text,
             let total:String = String(format: "%.2f", UserDefaults.standard.double(forKey: "total"))  {
-            totalPrice.text = totalPriceText + total
+            totalPrice.text = subTotalPriceText + total
+        }
+        
+        if let taxText = totalPrice.text {
```

Comment:
Nice work! this will display tax of current ordered by user based on subtotal price